ApiCompatTask/source/commandLineResult.ts

```diff
@@ -37,15 +37,19 @@ export default class CommandLineResult {
         `No differences were found between the assemblies`;
     }
 
-    public compattibilityResult = (): TaskResult => {
-        if (this._totalIssues) {
-            return TaskResult.Succeeded;
-        } else {
-            return this.failOnIssue ? TaskResult.Failed : TaskResult.SucceededWithIssues;
-        }      
+    public compatibilityResult = (): TaskResult => {

```

Comment:
Can we make this ternary operator more readable???

Reply:
@GasparAcevedoZainSouthworks Can be done, I'm testing the changes. Personally I believe adding `if`, `if else`, and `else` makes it harder to read at a glance and clutters it. But I understand why the change.